src/bun.js/webcore/body.zig

```diff
@@ -370,11 +371,20 @@ pub const Body = struct {
             }
         }
 
+        fn _lockedSize(this: *const Value) Blob.SizeType {
```

Comment:
Why is this a method on `Value` instead of `Locked`?  Why does it have an `_` in the name (it's a private method, that is unnecessary)

Reply:
i think i suggested the name, sorry